src/WebJobs.Script/Rpc/LanguageWorkerChannelManager.cs

```diff
@@ -48,8 +47,6 @@ public class LanguageWorkerChannelManager : ILanguageWorkerChannelManager
 
             _shutdownStandbyWorkerChannels = ScheduleShutdownStandbyChannels;
             _shutdownStandbyWorkerChannels = _shutdownStandbyWorkerChannels.Debounce(5000);
-            _rpcChannelReadySubscriptions = _eventManager.OfType<RpcWebHostChannelReadyEvent>()
-               .Subscribe(AddOrUpdateWorkerChannels);
```

Comment:
removing this subscription won't break anything?

Reply:
Nope. This subscription was specifically being used to add workerchannel to the cache.